docs/overview/running_effects.md

```diff
@@ -26,15 +26,15 @@ object MyApp extends zio.ZIOAppDefault {
 }
 ```
 
-If you are using a custom environment for your application, you will have to supply your environment to the effect (using `ZIO#provide`) before you return it from `run`, because `App` does not know how to supply custom environments.
+If you are using a custom environment for your application, you will have to supply your environment to the effect (using `ZIO#provide`) before you return it from `run`. `App` does not know how to supply custom environments.
 
 ## Default Runtime
 
-Most applications are not greenfield, and must integrate with legacy code, and procedural libraries and frameworks.
+Most applications are not greenfield, meaning they must integrate with legacy code and procedural libraries and frameworks.
 
 In these cases, a better solution for running effects is to create a `Runtime`, which can be passed around and used to run effects wherever required.
 
-ZIO contains a default runtime called `Runtime.default`. This `Runtime` bundles together production implementations of all ZIO modules (including `Console`, `System`, `Clock`, `Random`, `Scheduler`, and on the JVM, `Blocking`), and it can run effects that require any combination of these modules.
+ZIO contains a default runtime called `Runtime.default`. This `Runtime` bundles together production implementations of all ZIO modules (including `Console`, `System`, `Clock`, `Random`, `Scheduler`, and `Blocking` on the JVM), and it can run effects that require any combination of these modules.
```

Comment:
In ZIO 2.x, the `Blocking` service was removed, and also all default services were removed from `Runtime.default` and were built into the runtime system. 
So I think having this paragraph doesn't make sense anymore.

Reply:
Ah, okay. That considered, I agree. I can remove it. 🙃